lecture-notes/2-vcs.md

```diff
@@ -1758,22 +1758,114 @@ $
 If you had more remotes, they would also show up both in the configuration file
 and in the output of `git remote -v`.
 
-> As another aside, this convention of the path being `username/projectname` is
-> just that---convention. The server name and path name could be anything that
-> the server chooses. In GitHub's case, though, separating out by username
-> gives nice namespacing (same for the other forges).
+<!-- TODO(max): Figure out x-overflow -->
+<style>
+.sidenote {
+    font-size: 80%;
+    position: relative;
+    float: right;
+}
+/* Wide viewport */
+@media (min-width: 1400px) {
+    .sidenote {
+        clear: right;
+        margin-right: -23vw;
+        text-align: left;
+
+        top: -3rem;
+        width: 20vw;
+        margin-top: 1rem;
+    }
+}
+/* Narrow viewport */
+@media (max-width: 1400px) {
+    .sidenote {
+        text-align: right;
+        width: 100%;
+        margin: 1rem 0;
+        padding-left: 15%;
+    }
+}
+</style>
+
+<div class="sidenote" markdown=1>
+  As another aside, this convention of the path being `username/projectname` is
+  just that---convention. The server name and path name could be anything that
+  the server chooses. In GitHub's case, though, separating out by username
+  gives nice namespacing (same for the other forges).
+</div>
 
-So you have a remote now. You can tell Git to write a local ref to the remote
-by using the `push` subcommand:
+So you have a remote now. You can tell Git to write a local ref (in this case,
+the `main` branch) to the remote (in this case, `origin`) by using the `push`
+subcommand:
 
 ```console?prompt=$
+$ git push origin main
+...
+$
 ```
 
+To reduce the amount of typing, you can also use `--set-upstream`/`-u` one time
+to mark `origin/main` as the default destination for the local ref `origin`.
+This adds an entry in the `.git/config` file:
+
+```console?prompt=$
+$ git push -u origin main
+...
+$ cat .git/config
+...
+[branch "main"]
+	remote = origin
+	merge = refs/heads/main
+...
+$ # Now you can do `git push`
```

Comment:
Should also be a real example here

Reply:
i did it for -u origin main but not the latter, which will have the same output as the first